extension/content_script.js

```diff
@@ -1,6 +1,9 @@
 // Prevent the usage of undeclared variables.
 "use strict";
 
+// Is this used anywhere?
+// const { UI } = require("winjs");
+
```

Comment:
Is this used anywhere?

Reply:
I don't remember adding this. If ur not using it too, then safe to delete.